src/tiny-invariant.ts

```diff
@@ -1,9 +1,20 @@
 const isProduction: boolean = process.env.NODE_ENV === 'production';
 const prefix: string = 'Invariant failed';
 
-// Throw an error if the condition fails
-// Strip out error messages for production
-// > Not providing an inline default argument for message as the result is smaller
+/**
+ * Throw an error if the condition fails. Strip out error messages for production.
+ *
+ * @param condition A boolean condition - if falsey will thrown an error.
+ * @param message The message provided to accompany the invariant. No inline default argument for message as the result is smaller.
+ *
+ * @example
```

Comment:
I like the `@example`

Reply:
haha yeah. Playful but I think it helps to make it really clear how to use the fn :)